src/3rdparty/walkontable/src/table.js

```diff
@@ -360,6 +360,7 @@ class Table {
         // remove `innerBorderTop` and `innerBorderLeft` CSS classes to the DOM element. This happens
         // when there is a switch between rendering from 0 to N rows/columns and vice versa).
         wtOverlays.refreshAll();
+        wtOverlays.adjustElementsSize();
```

Comment:
I moved the `wtOverlays.adjustElementsSize()` method from all overlays to one place. From now on the Table class controls when the method is called.